.github/workflows/build-test-coverage.yml

```diff
@@ -0,0 +1,45 @@
+name: Build Test & Coverage
+on:
+    pull_request:
+    push:
+
+jobs:
+
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: "3.12"
```

Comment:
Is Python 3.12 sufficient for this? Seems like it would be better to matrix this workflow and test every supported minor version of Python. I'm not sure what the CircleCI job did but that's generally how most libraries approach testing.

Reply:
Addressed in the revised PR